README.md

```diff
@@ -312,47 +270,77 @@ The *Docker Server* is the full qualified hostname of your container registry.
 
 #### Create a HDI container and a secret
 
+This step is only required if you're using a BTP Trial account. If you're using a production or a free tier account then you can create HDI Container from Kyma directly by adding a [mapping to your Kyma namespace in your HANA Cloud Instance](https://blogs.sap.com/2022/12/15/consuming-sap-hana-cloud-from-the-kyma-environment/) and skip this step.
+
 ```
 bash ./scripts/create-db-secret.sh bookshop-db
 ```
 
 It will create a HDI container `bookshop-db` on your currently targeted Cloud Foundry space and creates a secret `bookshop-db` with the HDI container's credentials in your current Kubernetes namespace.
 
+Make the following changes to your _`chart/values.yaml`_.
+
+```diff
+srv:
+  bindings:
+    db:
+-     serviceInstanceName: hana
++     fromSecret: bookshop-db
+...
+
+hana-deployer:
+  bindings:
+    hana:
+-     serviceInstanceName: hana
++     fromSecret: bookshop-db
+
+...
+- hana:
+-   serviceOfferingName: hana
+-   servicePlanName: hdi-shared
+```
+
 ### Build
 
 **Build data base deployer image:**
 
-```
+```bash
 cds build --production
 
 pack build $YOUR_CONTAINER_REGISTRY/bookshop-hana-deployer \
      --path db \
      --buildpack gcr.io/paketo-buildpacks/nodejs \
-     --builder paketobuildpacks/builder:base
+     --builder paketobuildpacks/builder:base \
+     --env BP_NODE_RUN_SCRIPTS=""
 ```
 
 (Replace `$YOUR_CONTAINER_REGISTRY` with the full-qualified hostname of your container registry)
 
 
 **Build image for CAP service:**
 
-```
-mvn package
+```bash
+mvn clean package -DskipTests=true
 ```
 
-```
+```bash
 pack build $YOUR_CONTAINER_REGISTRY/bookshop-srv \
         --path srv/target/*-exec.jar \
         --buildpack gcr.io/paketo-buildpacks/sap-machine \
         --buildpack gcr.io/paketo-buildpacks/java \
         --builder paketobuildpacks/builder:base \
-        --env SPRING_PROFILES_ACTIVE=cloud
+        --env SPRING_PROFILES_ACTIVE=cloud \
+        --env BP_JVM_VERSION=17
```

Comment:
Is this required? Shouldn't this be the default, which is automatically detected by the buildpack from the Maven configuration?

Reply:
The buildpack's defaults changes frequently. It's better if we pass the flag here.